abilities-api.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Two Factor
+ *
+ * @package     WP_Abilities_API
+ * @author      WordPress.org Contributors
+ * @copyright   2020 Plugin Contributors
+ * @license     GPL-2.0-or-later
+ *
+ * @wordpress-plugin
+ * Plugin Name:       Abilities API
+ * Plugin URI:        https://github.com/WordPress/abilities-api
+ * Description:       Provides a framework for registering and executing AI abilities in WordPress.
+ * Requires at least: 6.7
+ * Version:           0.0.1
+ * Requires PHP:      7.2
+ * Author:            WordPress.org Contributors
+ * Author URI:        https://github.com/WordPress/abilities-api/graphs/contributors
+ * License:           GPLv2 or later
+ * License URI:       https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ * Text Domain:       abilities-api
```

Comment:
Overall makes sense. We need to figure out how to update the domain checks in the code that is expected to land in WP core as it should use the default domain like in Gutenberg.